### DISCUSSION TITLE
对推广ci的好消息

### DISCUSSION THREAD

**Discussion by MKStoler4096** [ClassIsland/ClassIsland#308] [展示台]
1. 我们信息老师给我了特权管理，现在基本上我管理着超2/3的班级（）
2. 除了因循守旧的，其余6个班都很喜欢ci（）
3. 祝ci越干越好✨

**Comment by Windows-Edge** [comment]
classisland的大旗已经插到了徐州市管道中学初中部！

**Comment by MKStoler4096** [comment]
不止如此，现在高一也用上了！

**Comment by Makitoid** [comment]
我们级部普及率5/17！
